packages/tabs/src/tabs-overflow.css

```diff
@@ -14,19 +14,24 @@ governing permissions and limitations under the License.
     inset: 0;
     width: 100%;
 
-    --sp-tabs-overflow-next-button-right: calc(
-        -1 * var(--spectrum-component-edge-to-text-100)
+    --sp-tabs-overflow-next-button-right: var(
+        --mod-sp-tabs-overflow-next-button-right,
+        calc(-1 * var(--spectrum-component-edge-to-text-100))
     );
-    --sp-tabs-overflow-previous-button-left: calc(
-        -1 * var(--spectrum-component-edge-to-text-100)
+    --sp-tabs-overflow-previous-button-left: var(
+        --mod-sp-tabs-overflow-previous-button-left,
+        calc(-1 * var(--spectrum-component-edge-to-text-100))
     );
     --sp-tabs-overflow-button-height: calc(
         var(--spectrum-tab-item-height-medium) -
             var(--spectrum-border-width-200)
     );
     --sp-tabs-overflow-button-size: var(--spectrum-tab-item-height-medium);
     --sp-tabs-overflow-icon-color: var(--spectrum-gray-800);
-    --sp-tabs-overflow-shadow-color: var(--spectrum-gray-100);
+    --sp-tabs-overflow-shadow-color: var(
+        --mod-sp-tabs-overflow-shadow-color,
+        var(--spectrum-gray-100)
+    );
```

Comment:
This is a temporary fix. and is against the mod inheritance property contract. Please reach out to the channel for better support from the spectrum-css team.